topo_processor/stac/collection.py

```diff
@@ -103,6 +112,42 @@ def get_bounding_boxes(self):
         union_poly = unary_union(polys)
         return [union_poly.bounds]
 
+    def get_linz_geospatial_type(self) -> str:
+        geospatial_type_set = set(x.linz_geospatial_type for x in self.items.values() if x.linz_geospatial_type)
+        if len(geospatial_type_set) != 1:
+            get_log().warning(f"Invalid 'linz:geospatial_type' collection='{self.title}'")
+            return "invalid geospatial type"
+        geospatial_type_str = geospatial_type_set.pop()
+        return geospatial_type_str
+
+    def get_linz_asset_summaries(self) -> Dict:
+        assets_checked: List[Asset] = []
+        dates_created: List[datetime] = []
+        dates_updated: List[datetime] = []
+
+        for item in self.items.values():
+            for asset in item.assets:
+                if not asset in assets_checked:
+                    if "created" in asset.properties:
+                        dates_created.append(asset.properties["created"])
+                        dates_updated.append(asset.properties["updated"])
+                    assets_checked.append(asset)
+
+        interval_created = get_min_max_interval(dates_created)
+        interval_updated = get_min_max_interval(dates_updated)
+
+        # to pass metadata-only validation as there are no assets to populate mandatory linz:asset_summaries
```

Comment:
I would suggest adding a `# TODO`, `# FIXME` to don't forget removing this when we'll do the cleanup to remove `validate` command, one day?